projects/dagster-slurm/dagster_slurm/local_runner.py

```diff
@@ -0,0 +1,50 @@
+import subprocess
+import shlex
+from pathlib import Path
+import os
+class LocalRunner:
```

Comment:
Can`t we completely re-use the default ray one for local? https://github.com/danielgafni/dagster-ray/blob/master/src/dagster_ray/core/pipes.py#L63 mode or the normal shell one for shell execution?